examples/lora/lorawan/atcmd/commands.go

```diff
@@ -337,6 +337,19 @@ func delay(setting string) error {
 
 func lw(setting string) error {
 	cmd := "LW"
+
+	param, val, hasComma := strings.Cut(setting, ",")
+	if hasComma {
+		if param == "NET" {
+			if val == "ON" {
+				lorawan.SetPublicNetwork(true)
+				println("SET PUBLIC NET")
```

Comment:
Doesn't `writeCommandOutput(cmd, setting)` output enough already?

Reply:
forgot this debug trace. Will fix